modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc

```diff
@@ -129,6 +129,18 @@ on the *Create Connector* page:
 | `Offset partition name`
 | `offset.partition.name`
 | Use to specify a custom offset partition name. If blank, the default partition name based on the connection details is used.
+
+| `Topic creation enabled`
+| `topic.creation.enable`
+| Specifies whether or not to allow automatic creation of topics. Default is enabled.
```

Comment:
Is the default value actually `enabled` or `true`?

Reply:
the actual is `topic.creation.enable=true`